### DISCUSSION TITLE
Next JS latest version application not working on IIS server

### DISCUSSION THREAD

**Discussion by afsajfej2342werw** [vercel/next.js#53851] [Help]
### Summary

403 - Forbidden: Access is denied.
You do not have permission to view this directory or page using the credentials that you supplied.

How to route to the main page of the domain or web.config setup.

**Comment by icyJoseph** [comment]
Are you using export mode? or is it just the normal `next start` server?

I tried helping someone with that a few months ago: https://github.com/vercel/next.js/discussions/51646#discussioncomment-6251291, I believe they ended up getting help from the support team of the site where they deploy their code.

**Comment by afsajfej2342werw** [comment]
I'm using 'next start',  deploying on live server can you suggest me about
config. for iis server deployment?

**Reply by icyJoseph** [reply]
Not really, I don't even have access to that OS.

Have you looked online? We can't be the only ones talking about this? 

- https://medium.com/@dipneupane/deploying-nextjs-application-on-windows-iis-server-cabdc22bccf8 talks about using iisnode?
- https://www.youtube.com/watch?v=HLsx0iraA-Y seems to expand on the previous article
- https://github.com/vercel/next.js/discussions/48441

**Comment by afsajfej2342werw** [comment]
Good Morning,


Thanks for your response. 🤗

Yes, I saw your video and article that's good.

I'm still trying many methods for solution..
